install/helm/openchoreo-control-plane/values.yaml

```diff
@@ -495,137 +503,186 @@ backstage:
+        type: sqlite
+        sqlitePath: "repository/database/thunderdb.db"
+        sqliteOptions: "_journal_mode=WAL&_busy_timeout=5000"
       runtime:
-        type: "sqlite"
-        path: "repository/database/runtimedb.db"
-        options: "_journal_mode=WAL&_busy_timeout=5000"
+        type: sqlite
+        sqlitePath: "repository/database/runtimedb.db"
+        sqliteOptions: "_journal_mode=WAL&_busy_timeout=5000"
+      user:
+        type: sqlite
+        sqlitePath: "repository/database/userdb.db"
+        sqliteOptions: "_journal_mode=WAL&_busy_timeout=5000"
+
+    # Cache configuration
+    cache:
+      disabled: false
+      type: "inmemory"
+      size: 1000
+      ttl: 3600
+      evictionPolicy: "LRU"
+      cleanupInterval: 300
+
+    # Token configuration
+    jwt:
+      issuer: "thunder"
+      validityPeriod: 3600
+      audience: "application"
+
+    # OAuth configuration
     oauth:
-      jwt:
-        issuer: "thunder"
-        validityPeriod: 3600
       refreshToken:
         renewOnGrant: false
         validityPeriod: 86400
+
+    # Flow configuration
     flow:
       graphDirectory: "repository/resources/graphs/"
       authn:
         defaultFlow: "auth_flow_config_basic"
-  runtimeConfig:
-    client:
-      base: "/signin"
-    server:
-      # port and httpOnly are auto-derived from global.devMode and TLS config
-      port: ""
-      httpOnly: ""
-  persistence:
-    database:
-      enabled: true
-      storageClassName: ""
-      accessMode: ReadWriteOnce
-      size: 500Mi
-      reclaimPolicy: Delete
-  backstagePortal:
+
+    # CORS configuration - Include both ingress hosts
+    cors:
+      allowedOrigins:
+        - "http://openchoreo.localhost:8080"
+        - "http://localhost:7007"
+
+  # Setup job configuration
+  setup:
     enabled: true
-    name: "Backstage"
-    description: "OpenChoreo Backstage Portal"
-    clientId: "openchoreo-backstage-client"
-    clientSecret: "backstage-portal-secret"
-    redirectUrls: []
-  userDetails:
-    userSchemaName: "engineer"
-    users:
+    backoffLimit: 3
+    ttlSecondsAfterFinished: 86400  # 24 hours
+    preserveJob: true
+    debug: false
+    resources:
+      limits:
+        cpu: 500m
+        memory: 256Mi
+      requests:
+        cpu: 250m
+        memory: 128Mi
+    # Pass Backstage OAuth credentials to setup job via environment variables
+    # Note: These values are also stored in backstage-secrets for Backstage to use
+    env:
+      - name: BACKSTAGE_CLIENT_ID
+        value: "openchoreo-backstage-client"
+      - name: BACKSTAGE_CLIENT_SECRET
+        value: "backstage-portal-secret"  # Must match backstage.auth.clientSecret
+      - name: BACKSTAGE_REDIRECT_URLS
+        # IMPORTANT: Must match backstage.auth.redirectUrls (comma-separated)
+        value: "http://openchoreo.localhost:8080/api/auth/default-idp/handler/frame,http://localhost:7007/api/auth/default-idp/handler/frame"
```

Comment:
shall we remove these default values from here and make sure these are auto generated based on `base domain` ? we can have them as commented for reference in case for overides